src/components/common/form.js

```diff
@@ -58,6 +57,40 @@ const Textarea = ({
   </>
 )
 
+const InputDefinedLength = ({
+  label,
+  type,
+  id,
+  name,
+  isRequired,
+  detailText,
+  onChange,
+  maxLength,
+  className,
+}) => (
+  <div className={formStyles.inputDefinedLength}>
+    <div>
+      <FormLabel htmlFor={id} isRequired={isRequired}>
+        {label}
+      </FormLabel>
+      <input
+        name={name}
+        id={id}
+        type={type}
+        aria-required={isRequired}
+        onChange={onChange}
+        maxLength={maxLength}
+        className={`${className} ${
```

Comment:
I'm seeing a lot of inputs with class of `undefined null` - Let's import the `classnames` module and do:

```
className={classnames(className, detailText ? formStyles.inputDescription : false)}
```

The classnames module will just skip empty values and should return a cleaner class name.